Pagination for Snowflake objects (#6445)
fishtown-analytics/docs.getdbt.com: website/docs/reference/resource-configs/snowflake-configs.md

  >- DDL operations.
  >- DML operations (for tables only).
  >- Background maintenance operations on metadata performed by Snowflake.

<VersionBlock firstVersion="1.9">

## Pagination for object results

> By default, when dbt encounters a schema with up to 100k objects, it will paginate the results from `show objects` at 10k per page for up to 10 pages.

Reviewer: ```suggestion
By default, when dbt encounters a schema with up to 100,000 objects, it will paginate the results from `show objects` at 10,000 per page for up to 10 pages.
```

> Environments that have more than 100k objects in a schema can customize the number of results per page and the page limit with the following flags in the `dbt_project.yml`:

Reviewer: ```suggestion
Environments that have more than 100,000 objects in a schema can customize the number of results per page and the page limit with the following flags in the `dbt_project.yml`:
```

> Environments that have more than 100,000 objects in a schema can customize the number of results per page and the page limit with the following flags in the `dbt_project.yml`:

Author: ```suggestion
Environments that have more than 100,000 objects in a schema can customize the number of results per page and the page limit with the following [flags](/reference/global-configs/about-global-configs) in the `dbt_project.yml`:
```

> Environments that have more than 100,000 objects in a schema can customize the number of results per page and the page limit with the following [flags](/reference/global-configs/about-global-configs) in the `dbt_project.yml`:

Reviewer 2: ```suggestion
Environments with more than 100,000 objects in a schema can customize the number of results per page and the page limit using these [flags](/reference/global-configs/about-global-configs) in the `dbt_project.yml`:
```

Author: I'll split the difference with you 

```suggestion
Environments with more than 100,000 objects in a schema can customize the number of results per page and the page limit using the following [flags](/reference/global-configs/about-global-configs) in the `dbt_project.yml`:
```

> - `list_relations_per_page` &mdash; The number of relations on each page (Max 10k as this is the most Snowflake allows).

Reviewer: do we have links to how flags are done in docs?

- `list_relations_page_limit` &mdash; The maximum number of pages to include in the results.

> For example, if you wanted to include 10k objects per page and include up to 100 pages (1M objects), configure the flags as follows:

Reviewer: ```suggestion
For example, if you wanted to include 10,000 objects per page and include up to 100 pages (1 million objects), configure the flags as follows:
```
